codacy/values.yaml

```diff
@@ -23,6 +23,14 @@ global:
     workerManager:
       url: "codacy-worker-manager"
 
+  github: {}
+    # if you set this provider configuration, you must save it on the admin/integrations ui.
+    # enabled: "false"
+    # app:
+    #  appId: "1234"
+    #  TODO: Add link to documentation
```

Comment:
Remove this and create a task to track that its done when the docs are available